python/cudnn/gemm/cutedsl/grouped/dglu/moe_blockscaled_grouped_gemm_dglu_rubin.py

```
@@ -3608,27 +3309,25 @@ def kernel(
                             cute.make_layout(128),
                             cute.make_layout(self.mma_tiler[1] * 2),
                         )
-                        gD_sub_loop = cute.local_tile(
-                            real_d, d_epilogue_subtile, (None, None, None)
-                        )
+                        gD_sub_loop = cute.local_tile(real_d, d_epilogue_subtile, (None, None, None))
                         tCgD_mnl_loop = thr_copy_t2r.partition_D(gD_sub_loop)
```

Ruff (0.16.1)

[error] Undefined name `thr_copy_t2r`

(F821)